src/app/pages/events/sfd-2024/components/presentation/card-presentation/card-presentation.component.html

```diff
@@ -107,71 +105,13 @@ <h3 class="z-20">{{data.aditional[0].description}}</h3>
         <!-- doesn't work with the for in production but work in local jeje -->
         <!-- @for (item of [0,1,2,3,4,5,6,7,8,9,10]; track $index) {
         <div
-            class="w-[{{20 + (30*$index)}}%] aspect-square rounded-full border-4 border-[#cab9b6] absolute opacity-10 [transform:_translate(-50%,_-50%)]">
-        </div>
+            class="w-[{{20 + (30*$index)}}%] aspect-square rounded-full border-4 border-[#cab9b6] absolute opacity-10 [transform:_translate(-git@github.com:Backbone-UTP/main-page.git50%,_-50%)]">
```

Comment:
Creo que esto no debería de ir aquí o si?

Reply:
Incluso ya se puede eliminar ese comentario ya que trataba de hacer los circulos con un for.

Reply:
Si, se puede eliminar, olvide hacerlo.